modules/minimum-required-permissions-upi-azure.adoc

```diff
@@ -0,0 +1,241 @@
+* `Microsoft.Compute/snapshots/delete`
+* `Microsoft.Compute/virtualMachines/delete`
+* `Microsoft.Compute/virtualMachines/powerOff/action`
+* `Microsoft.Compute/virtualMachines/read`
+* `Microsoft.Compute/virtualMachines/write`
+* `Microsoft.Compute/virtualMachines/deallocate/action`
+====
+
+.Required permissions for creating identity management resources
+[%collapsible]
+====
+* `Microsoft.ManagedIdentity/userAssignedIdentities/assign/action`
+* `Microsoft.ManagedIdentity/userAssignedIdentities/read`
+* `Microsoft.ManagedIdentity/userAssignedIdentities/write`
+====
+
+.Required permissions for creating network resources
+[%collapsible]
+====
+* `Microsoft.Network/dnsZones/A/write`
+* `Microsoft.Network/dnsZones/CNAME/write`
+* `Microsoft.Network/dnszones/CNAME/read`
+* `Microsoft.Network/dnszones/read`
+* `Microsoft.Network/loadBalancers/backendAddressPools/join/action`
+* `Microsoft.Network/loadBalancers/backendAddressPools/read`
+* `Microsoft.Network/loadBalancers/backendAddressPools/write`
+* `Microsoft.Network/loadBalancers/read`
+* `Microsoft.Network/loadBalancers/write`
+* `Microsoft.Network/networkInterfaces/delete`
+* `Microsoft.Network/networkInterfaces/join/action`
+* `Microsoft.Network/networkInterfaces/read`
+* `Microsoft.Network/networkInterfaces/write`
+* `Microsoft.Network/networkSecurityGroups/join/action`
+* `Microsoft.Network/networkSecurityGroups/read`
+* `Microsoft.Network/networkSecurityGroups/securityRules/delete`
+* `Microsoft.Network/networkSecurityGroups/securityRules/read`
+* `Microsoft.Network/networkSecurityGroups/securityRules/write`
+* `Microsoft.Network/networkSecurityGroups/write`
+* `Microsoft.Network/privateDnsZones/A/read`
+* `Microsoft.Network/privateDnsZones/A/write`
+* `Microsoft.Network/privateDnsZones/A/delete`
+* `Microsoft.Network/privateDnsZones/SOA/read`
+* `Microsoft.Network/privateDnsZones/read`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/read`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/write`
+* `Microsoft.Network/privateDnsZones/write`
+* `Microsoft.Network/publicIPAddresses/delete`
+* `Microsoft.Network/publicIPAddresses/join/action`
+* `Microsoft.Network/publicIPAddresses/read`
+* `Microsoft.Network/publicIPAddresses/write`
+* `Microsoft.Network/virtualNetworks/join/action`
+* `Microsoft.Network/virtualNetworks/read`
+* `Microsoft.Network/virtualNetworks/subnets/join/action`
+* `Microsoft.Network/virtualNetworks/subnets/read`
+* `Microsoft.Network/virtualNetworks/subnets/write`
+* `Microsoft.Network/virtualNetworks/write`
+====
+
+.Required permissions for checking health of resources
+[%collapsible]
+====
+* `Microsoft.Resourcehealth/healthevent/Activated/action`
+* `Microsoft.Resourcehealth/healthevent/InProgress/action`
+* `Microsoft.Resourcehealth/healthevent/Pending/action`
+* `Microsoft.Resourcehealth/healthevent/Resolved/action`
+* `Microsoft.Resourcehealth/healthevent/Updated/action`
+====
+
+.Required permissions for creating resource group
```

Comment:
```suggestion
.Required permissions for creating resource groups
```

Reply:
@bscott-rh Here the singular form of `group` was used intentionally because we can create only a single resource group in the cluster. cc @chiragkyal

Reply:
But I will double check with @chiragkyal if resource groups can be used

Reply:
AFAIK, we only need one resource group to deploy an OCP cluster. Hence I suggested to use `group` in place of `groups`. cc @jinyunma

Reply:
For UPI, it's true. 
But for IPI, if user deploys on existing vNet, then the vNet related resources can be allocated in another resource group.

Reply:
@jinyunma, even if the user deploys on existing vNet, is there any possibility that the installer will handle (create/delete) two resource groups? I am not sure about that.

Reply:
no, installer is only responsible for destroying the resource group where installer created, there is only one resource group installer handling.

Reply:
Sounds good. In that case, it should be `for creating a resource group`.

Reply:
@xenolinux looks like this still needs to be updated to `a resource group`.